web/directory/urls.py

```diff
@@ -1,37 +1,41 @@
+from drf_spectacular.views import SpectacularAPIView, SpectacularSwaggerView, SpectacularRedocView
+
 
 urlpatterns = [
-    path('data', views.data, name='data'),
-    path('coops/no_coords', views.coops_wo_coordinates, name='coops_wo_coordinates'),
-    path('coops/unapproved', views.unapproved_coops, name='unapproved_coops'),
-    path('coops/', views.CoopList.as_view()),
-    path('coops/<int:pk>/', views.CoopDetail.as_view()),
-    path('coops/all/', views.CoopListAll.as_view()),
-    path('people/', views.PersonWithCoopList.as_view()),
-    path('people/<int:pk>/', views.PersonDetail.as_view()),
-    path('users/', views.CreateUserView.as_view()),
-    path('predefined_types/', views.CoopTypeList.as_view()),
-    path('coop_types/', views.CoopTypeList.as_view()),
-    path('countries/', views.CountryList.as_view()),
-    path('states/<country_code>', views.StateList.as_view()),
-    path('login', views.signin),
-    path(settings.LOGOUT_PATH, views.signout),
-    path('user_info', views.user_info),
-    path('reset_password', views.ResetPasswordView.as_view(template_name='../templates/users/password_reset.html'), name='reset_password'),
-    path('password-reset-confirm/<uidb64>/<token>/',
-         auth_views.PasswordResetConfirmView.as_view(template_name='users/password_reset_confirm.html'),
-         name='password_reset_confirm'),
-    path('password-reset-complete/',
-         auth_views.PasswordResetCompleteView.as_view(template_name='users/password_reset_complete.html'),
-         name='password_reset_complete'),
-]
 
-urlpatterns = format_suffix_patterns(urlpatterns)
+    path('api/v1/schema/', SpectacularAPIView.as_view(), name='schema'),
+    path('api/v1/docs/', SpectacularSwaggerView.as_view(url_name='schema'), name='swagger-ui'),
+    path('api/v1/admin/', admin.site.urls),
+
+    path('api/v1/token/', TokenObtainPairView.as_view(), name='token_obtain_pair'),
+    path('api/v1/token/refresh/', TokenRefreshView.as_view(), name='token_refresh'),
+
+    path('api/v1/users/', views.UserList.as_view(), name='user-list'),
+    path('api/v1/users/<int:pk>/', views.UserDetail.as_view(), name='user-detail'),
+    path('api/v1/register/', views.CreateUserView.as_view(), name='register'),
+    path('api/v1/password_reset/', views.PasswordResetRequestView.as_view(), name='password-reset-request'),
+    path('api/v1/password-reset-confirm/<str:uidb64>/<str:token>/', views.PasswordResetConfirmView.as_view(), name='password-reset-confirm'),
 
+    path('api/v1/coops/', views.CoopList.as_view(), name='coop-list'),
+    path('api/v1/coops/<int:coop_public_id>/', views.CoopDetail.as_view(), name='coop-detail'),
+    path('api/v1/coops/no_coords', views.CoopsNoCoords.as_view(), name='coop-no-coords'),
+    path('api/v1/coops/unapproved', views.CoopsUnapproved.as_view(), name='coop-unapproved'),
+    path('api/v1/coops/csv/', views.CoopCSVView.as_view(), name='data'),
```

Comment:
I wasn't able to work out what this URL does - could you walk me through it when you have a chance?

Reply:
- api/v1/coops/<int:coop_public_id>/: Returns the live coop record for a given coop_public_id
- api/v1/coops/no_coords: Admin view. Returns all coops without coordinates (meaning the location service failed on it). Carry over from the previous version. 
- api/v1/coops/unapproved: Admin view. Returns all coops proposals with a PENDING status. Can also be retrieved via CoopProposalList. Not sure if we need both. Carry over from the previous version. 
- api/v1/coops/csv/: Used by OpenStreetMap to fetch data to display on the map. Replaces the old "data" endpoint.